packages/fxa-graphql-api/src/lib/auth.ts

```diff
@@ -17,7 +17,16 @@ export class SessionTokenAuth {
     try {
       return await this.authClient.sessionStatus(sessionToken);
     } catch (err) {
-      throw new AuthenticationError('Invalid session token');
+      if (err.code === 400 || err.code === 401) {
```

Comment:
auth client error codes are defined here: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#defined-errors

Reply:
400 is a bad request, maybe we should always throw ApolloError if we can customize it better?

Reply:
The slight abuse of the 400 error code is pretty clear from the auth server docs that I linked above. Seems weird that "unverified account" throws a 400, not a 401, but here we are.

We could instead always throw ApolloError, but then we'll lose the "redirect on auth error" logic that we have on the front-end, or have to hyper-specialize it to various auth server `errno` values :-\

Reply:
This seems fine. The answer to any 4xx error on `/session/status` is reauth anyway